roles/confluent.control_center/molecule/rbac-custom-certs-1way-mds/prepare.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Install and configure OpenLDAP
+  hosts: ldap_server
+  tasks:
+    - import_role:
+        name: confluent.ldap
+
+    - name: Install OpenSSL Rsync and Java
+      yum:
+        name: "{{item}}"
+        state: present
+      loop:
+        - openssl
+        - rsync
+        - java-1.8.0-openjdk
+
+    - name: Create SSL Certificate Generation Directory
```

Comment:
I think these steps can be removed bc cp-ansible now creates the mds certs for you, would be better to update this scenario to test out providing your own mds certs!

Reply:
also the jenkins job failed on this one, so fix that

Reply:
resolving in this pr: https://github.com/confluentinc/cp-ansible/pull/247